src/backend/scripts/setup_localdata.py

```diff
@@ -510,7 +503,7 @@ def create_samples(
     for suffix in range(num_successful):
         _ = create_sample(session, group, user, pathogen, location, suffix)
     for suffix in range(num_failures):
-        _ = create_sample(session, group, user, pathogen, location, suffix, True)
+        _ = create_sample(session, group, user, pathogen, location, suffix)
```

Comment:
We can probably eliminate this loop, right? And eliminate the `num_failures` parameter in the `create_samples()` function it lives in. We would just have to modify line 565 slightly.

Reply:
yes thank you

Reply:
i accidentally missed that